vars/capsulePipelineWrapper.groovy

```diff
@@ -145,10 +145,11 @@ def call() {
                     stage('Sync remote state to local') {
                         steps {
                             sh """
-                                mkdir -p ${env.CONFIG_HOME}
-                                aws s3 sync ${env.S3_CONFIG_HOME}/ ${env.CONFIG_HOME}/
+                            set -x;
```

Comment:
FYI you can define that you use bash (by default bash is not used).
Eg https://github.com/vegaprotocol/jenkins-shared-library/blob/main/vars/pipelineDevnetDeploy.groovy#L166

Reply:
Ahh sorry set -x is just testing. Let me remove it